spec/integration/add_document_associations_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Add document associations", type: :feature do
+  scenario "A document has associations added" do
+    given_there_is_a_document_with_associations
+    when_i_visit_the_document_page
+    then_i_can_navigate_to_associations
+    when_i_add_some_associations
+    then_i_can_view_the_associations
+  end
+
+  def given_there_is_a_document_with_associations
+    @document = create :document, document_type: "press_release"
```

Comment:
Perhaps create a factory that is specific about what kind of document type we want here, instead of the document type:

```rb
create(:document, :with_a_topical_event_association)
```

Reply:
Yup sure, guess that'd mean feature tests for each link type but that's probably cool - albeit a touch verbose.